blog/2025-11-07-calculated-state-capability-attributes-removed.md

```diff
@@ -0,0 +1,9 @@
+---
+author: Artur Pragacz
+authorURL: https://github.com/arturpragacz
+title: "The capability_attributes field has been removed from CalculatedState"
+---
+
+The `capability_attributes` field has been removed from `CalculatedState`. Capability attributes are still included in all attributes, which remain available in `CalculatedState`.
```

Comment:
Please add some context that `CalculatedState` is used in `Entity._async_calculate_state`.